contracts/interfaces/IHATVaultsRegistry.sol

```diff
@@ -56,15 +56,6 @@ interface IHATVaultsRegistry {
         uint256 claimFee;  
     }
 
-    // How to divide the HAT part of bounties, in percentages (out of {HUNDRED_PERCENT})
```

Comment:
good call. Should also remove the reference in the docstring in IHATVAult and in roles.md

Reply:
right, thanks!

Reply:
done